content/cpp/concepts/inheritance/terms/multiple-inheritance/multiple-inheritance.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: 'Multiple Inheritance'
+Description: 'Enables classes to inherit characteristics from more than one class.'
+Subjects:
+  - 'Computer Science'
```

Comment:
We can also add 'Code Foundations' as a subject here, check the full list of subjects [here](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).

```suggestion
  - 'Computer Science'
  - 'Code Foundations'
```